intgtest/intgSpec.js

```diff
@@ -310,7 +310,7 @@ describe('Vantiq SDK Integration Tests', function() {
         })
         .then(function() {
             should.exist(resp);
-            resp.headers['X-Request-Id'].should.equal('/topics//test/topic');
+            resp.headers['X-Request-Id'].should.equal('/topics/test/topic');
```

Comment:
why has this changed to not have the double-slash?  
I thought the leading slash is part of the topic-name.
Is that a compatibility issue with existing users of the SDK?

Reply:
Currently all other subscriptions and event paths use the /topics/my/topic/name model. As it currently is, we'll still allow users to subscribe with the double slash topic name if they'd like. However, the returned event path/RequestId will use the single slash. We'll also assume that all users subscribing to persistent subscriptions will use the single slash name

Reply:
would be good to add a comment like this to the test-case change... maybe even reference the ag2rs change # that got rid of the double-slash.  (assuming it was an ag2rs change & not an SDK change...)